com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumeBakingProcessSettings.cs

```diff
@@ -19,8 +19,12 @@ internal struct ProbeDilationSettings
     internal struct VirtualOffsetSettings
     {
         public bool useVirtualOffset;
-        public float outOfGeoOffset;
-        public float searchMultiplier;
+        [Range(0f, 1f)] public float outOfGeoOffset;
+        [Range(0f, 2f)]public float searchMultiplier;
```

Comment:
Missing space here :) 


Also generally I think these settings might be a tad too advanced for users. I would add them under an advanced foldout in the editor.

Reply:
Which ones do you want under an advanced foldout? All of them?

Reply:
Yes probably